eva_submission/eload_ingestion.py

```diff
@@ -1,20 +1,16 @@
-import string
 import subprocess
 
-from bs4 import BeautifulSoup
 from ebi_eva_common_pyutils import command_utils
 from ebi_eva_common_pyutils.config import cfg
 from ebi_eva_common_pyutils.mongo_utils import get_mongo_connection_handle
+from ebi_eva_common_pyutils.pg_utils import get_all_results_for_query
+from lxml import etree
```

Comment:
Awesome! Sorry for having been a stickler about all this... It is just to minimize external dependencies (particularly big ones like bs4)...

Reply:
no worries, please do point these things out!  it helps me remember to think about it next time.

... though as we're getting rid of the ENA call i don't think we need either lxml or bs4 anymore :laughing: